code/modules/hydroponics/hydroponics.dm

```diff
@@ -1170,6 +1170,9 @@
 /obj/machinery/hydroponics/soil/CtrlClick(mob/user)
 	return //Soil has no electricity.
 
+/obj/machinery/hydroponics/soil/Destroy()
+	new /obj/item/stack/ore/glass(drop_location(), 3)
+	return ..()
```

Comment:
> Put this in `deconstruct`, not `destroy` Additionally, `attackby_secondary` for shovels should `deconstruct()` and not `qdel(src)`
> 
> Side effects in `destroy` are no-good

I had it in deconstruct but then moved to destroy to make it spawn piles when blown up. I thought it's a nice touch. 

![image](https://user-images.githubusercontent.com/3625094/235295221-7476650b-56d9-47cb-895f-ce556c22c24b.png)

Is it okay if I move these into contents from destroy? Lke machines drop parts, it will drop sand.